### PR TITLE
Update peer dependencies to include Svelte 5

### PR DESCRIPTION
**Pull request by attilabuturla**

### CODE REVIEW

**Comment by benmccann** [comment]
Can you please remove the `package-lock.json` file and run `pnpm install` to update the `pnpm-lock.yaml` file instead?

**Comment by attilabuturla** [comment]
> Can you please remove the `package-lock.json` file and run `pnpm install` to update the `pnpm-lock.yaml` file instead?

@benmccann  Done. Note that there were no changes to the `pnpn-lock.yaml` file even with a `pnpm install`.

File: .changeset/brave-papayas-push.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/svelte-json-tree': patch
+---
+
+Update peer dependency to support Svelte 5.
```
Comment (benmccann) [inline .changeset/brave-papayas-push.md]:
```suggestion
fix: update peer dependencies to support Svelte 5
```

File: .changeset/thin-suns-stare.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/svelte-json-tree': patch
+---
+
+Update peer dependencies to support Svelte 5.
```
Comment (benmccann) [inline .changeset/thin-suns-stare.md]: it looks like there are two changesets on this PR. please remove the duplicate

**Review by benmccann — APPROVED** [review]

**Comment by Rich-Harris** [comment]
released 2.2.1 with the fix — thanks!

**Pull request merged**